about.html

```diff
@@ -72,7 +74,8 @@ <h2 class="partner__title section-title"> Partner</h2>
         </section>
     </main>
     <footer>
-        <p>Capstone project 1 - made by: <br>Andres Agudelo Guzman.</p>
+        <p>Capstone project 1 - made by: <br>Andres Agudelo Guzman.<br> *This is a fantasy music festival web page, made for educational purposes*</p>
     </footer>
```

Comment:
- [ ] Make sure that the fonts (background color) for your footer section on the about page fit the original [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).